Rubberduck.Parsing/Rubberduck.Parsing.csproj

```
@@ -13,6 +13,8 @@
     <WarningLevel>1</WarningLevel>
     <UseVSHostingProcess>true</UseVSHostingProcess>
     <NoWarn>1701;1702;1591;4011;1001;7035;1053</NoWarn>
+    <DocumentationFile>Rubberduck.Parsing.xml</DocumentationFile>
+    <OutputPath />
```

this declaration may be messing with the output path overwriting we're doing in the base csproj. https://github.com/rubberduck-vba/Rubberduck/blob/433de5a56f18de8259851b154d2bf2dbb33ba1fe/RubberduckBaseProject.csproj#L36-L40

The "Proper" way to do this would be

```suggestion
    <UnifyOutputPath>true</UnifyOutputPath>
```

---

Thanks, trying that now. Not sure why CodeAnalysis.csproj doesn't have that?

---

Worked!